test/DGmethods/advection_diffusion/pseudo1D_advection_diffusion.jl

```diff
@@ -30,6 +33,27 @@ const output = parse(Bool, lowercase(get(ENV,"JULIA_CLIMA_OUTPUT","false")))
 
 include("advection_diffusion_model.jl")
 
+# provide default units
+space_unit(::AdvectionDiffusion) = u"m"
+time_unit(::AdvectionDiffusion) = u"s"
```

Comment:
Can we just use the methods in `aliases.jl` here?

Reply:
(and same in other files)

Reply:
Sweet! All manual specification of units have been replaced with unit aliases.